target.cpp

```diff
@@ -336,8 +336,10 @@ void TargetClient::OnPaint(wxPaintEvent& WXUNUSED(evt))
     }
 
     // Draw labels
-    dc.DrawText(_("RA"), leftEdge, center.y - 15);
-    dc.DrawText(_("Dec"), center.x - 35, topEdge - 3);
+    int d_width = 10, d_height = 10;
+    dc.GetTextExtent(_("0"), &d_width, &d_height);

```

Comment:
we do not want this string translated
```suggestion
    dc.GetTextExtent(_T("0"), &d_width, &d_height);
```

Reply:
Ok